### PR TITLE
Add -render-type-content option to render full type content instead of an ellipsis (`{ ... }`)

### PR DESCRIPTION
**Pull request by alexandregv**
Adds a `-render-type-content` bool flag which allows to control wether you want to replace types content with an ellipsis or not.  
It defaults to false, to keep the current behavior.  

Example with `-render-type-content=true`:
```go
type Config struct { ... }
```
becomes
```go
type Config struct {
    Username      string `json:"username"`
    Password      string `json:"password"`
}
```

### CODE REVIEW

File: cmd/goreadme/main.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ func init() {
 	flag.StringVar(&cfg.ImportPath, "import-path", "", "Override package import path.")
 	flag.StringVar(&cfg.Title, "title", "", "Override readme title. Default is package name.")
 	flag.BoolVar(&cfg.RecursiveSubPackages, "recursive", false, "Load docs recursively.")
+	flag.BoolVar(&cfg.Ellipsis, "ellipsis", true, "If 'types' is specified, replace types content with an ellipsis.")
```
Comment (posener) [inline cmd/goreadme/main.go:51]: Maybe replace ellipsis with render-type-content which is  default to false.

**Review by posener — CHANGES_REQUESTED** [review]

File: cmd/goreadme/main.go (hunk 1 above)
Comment (alexandregv) [reply cmd/goreadme/main.go]: Indeed, seems clearer. Just pushed the change.

**Comment by posener** [comment]
Maybe add a test? Also update pr text? Thanks!

**Comment by alexandregv** [comment]
Just added a test, a changed PR text.

**Review by posener — APPROVED** [review]

**Pull request merged**